stages/org.osbuild.ovf

```diff
@@ -65,25 +68,46 @@ OVF_TEMPLATE = """<?xml version="1.0"?>
         <rasd:ResourceSubType>VirtualSCSI</rasd:ResourceSubType>
         <rasd:ResourceType>6</rasd:ResourceType>
       </Item>
-      <Item>
-        <rasd:Address>0</rasd:Address>
-        <rasd:Description>IDE Controller</rasd:Description>
-        <rasd:ElementName>VirtualIDEController 0</rasd:ElementName>
-        <rasd:InstanceID>4</rasd:InstanceID>
-        <rasd:ResourceType>5</rasd:ResourceType>
-      </Item>
```

Comment:
I guess it would probably be fine for CoreOS disk images to have an IDE controller. So we could share that?

Reply:
I'm happy to try the path of removing it by default I think.. I'll ask around to see if we (CoreOS) somehow rely on this.

Reply:
oh nevermind.. we already don't have that by default. so yeah. I can add a commit to remove that if you're OK with it.